docs/guides/templating.md

```diff
@@ -2,48 +2,54 @@
 
 With External Secrets Operator you can transform the data from the external secret provider before it is stored as `Kind=Secret`. You can do this with the `Spec.Target.Template`. Each data value is interpreted as a [golang template](https://golang.org/pkg/text/template/).
 
+!!! note
+
+    **.'spec.data.secretkey'** cannot contain **'-'** (dashes), consider using camelcase when defining the secret keys.
+
+    Example: service-account-token -> serviceAccountToken
 
 ## Helm
 
 When installing ExternalSecrets via `helm`, the template must be escaped so that `helm` will not try to render it. The most straightforward way to accomplish this would be to use backticks ([raw string constants](https://pkg.go.dev/text/template#hdr-Examples)):
 
 ```yaml
-{% include 'helm-template-v2-escape-sequence.yaml' %}
+{ % include 'helm-template-v2-escape-sequence.yaml' % }
```

Comment:
these whitespaces unfortunately break the template 
![2023-11-02T19:18:59,977502962+01:00](https://github.com/external-secrets/external-secrets/assets/1709030/f839d969-eae4-46d4-90c6-e6ecbfcbcf89)

Reply:
Im keeping well, Hope the Same. 

That was my auto-formatter being a a little over zealous 🤣